apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/index.tsx

```diff
@@ -142,13 +144,13 @@ const TransactionConfirm = ({
     status,
   });
 
-  const typeTransaction: string = useMemo(
+  const typeTransaction: string | undefined = useMemo(
     () =>
       (
         fields.find(
           (field: { label: string }) => field.label && field.label === "Type",
         ) as DeviceTransactionField & { value: string }
```

Comment:
Same comment as above for the `as` keyword

Reply:
🧹 https://github.com/LedgerHQ/ledger-live/pull/6205/commits/7082400185baebf65ec655faed2bba8bef4f0f1e